cypress/support/pages/login_management/pageLoginManagement.js

```diff
@@ -11,7 +11,7 @@ class Login_Management {
   static #emailInputBox = '[data-testid="username-email"]'
   static #passwordField = '[data-testid = "password-email"]'
   static #notificationBannerField = '[data-testid="notification"]'
-  static #loginFormSelector = 'form.login-form'
+  static #loginFormSelector = '[data-testid = "login-loginform-standard"]'
   static #inputFieldInvalidPseudoSelector = 'input:invalid'
```

Comment:
can we assign also in this element data-testid ?

Reply:
Which element do you mean exactly?

Reply:
in line #15 ->  static #inputFieldInvalidPseudoSelector = 'input:invalid'

Reply:
This is not a normal element in the DOM, so we can't assign a data-testid